style.css

```diff
@@ -71,8 +71,6 @@ header {
 }
 
 .stackbar {
-  display: flex;
-  gap: 15px;
   list-style-type: none;
 }
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible, The spacing, should be similar to the Figma template.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212924546-b0da0a08-f425-4f2c-9619-f4b3f7c7e352.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212925410-73a84c14-933b-440c-8407-c753cd9e75d6.png)